doc/16-upgrading-icinga-2.md

```diff
@@ -7,6 +7,22 @@ Specific version upgrades are described below. Please note that version
 updates are incremental. An upgrade from v2.6 to v2.8 requires to
 follow the instructions for v2.7 too.
 
+## Upgrading to v2.8.2 <a id="upgrading-to-2-8-2"></a>
+
+With version 2.8.2 the location of settings formerly found in `/etc/icinga2/init.conf` has changed. They are now
+located in the sysconfig (`/etc/sysconfig/icinga2` on most systems). The `init.conf` has been removed and its settings
```

Comment:
The location of the file should perhaps be `/etc/sysconfig/icinga2` on RPM-based systems and `/etc/default/icinga2` on DPKG-based systems.

Reply:
It installed to `/etc/sysconfig/icinga2` on my buster. But that might have to do with it being a manual build. I'll add it!